src/app/shared/components/loading/loading.component.scss

```diff
@@ -0,0 +1,4 @@
+::ng-deep .preloader.mat-progress-spinner circle,
```

Comment:
`::ng-deep` is deprecated, it'd be good to find a better way...

Reply:
poprawione